spec/request/mhv_accounts_request_spec.rb

```diff
@@ -147,146 +143,134 @@
     end
   end
 
-  context 'without T&C acceptance' do
-    it_behaves_like 'a successful GET #show', account_state: 'needs_terms_acceptance', account_level: nil
-    it_behaves_like 'a failed POST #create', http_status: :forbidden,
-                                             message: V0::MhvAccountsController::CREATE_ERROR
-    it_behaves_like 'a failed POST #upgrade', http_status: :forbidden,
-                                              message: V0::MhvAccountsController::UPGRADE_ERROR
-    it_behaves_like 'ssn mismatch'
-    it_behaves_like 'non va patient'
-  end
-
-  context 'with T&C acceptance' do
-    before { create(:terms_and_conditions_acceptance, terms_and_conditions: terms, user_uuid: user.uuid) }
+  it_behaves_like 'ssn mismatch'
```

Comment:
Similarly, all of the following changes are just shifting blocks out of the T&C contexts.